buildpipeline/linux.groovy

```diff
@@ -52,7 +52,6 @@ simpleDockerNode('microsoft/dotnet-buildtools-prereqs:rhel7_prereqs_2') {
                                      'Debian.87.Amd64.Open',
                                      'Ubuntu.1404.Amd64.Open',
                                      'Ubuntu.1604.Amd64.Open',
-                                     'Ubuntu.1610.Amd64.Open',
```

Comment:
@stephentoub  file a dotnet/core-eng issue if you'd like and we can add that, there are already images on Azure for the 17.10-DAILY SKU so it's not much work, but in CI this may prove noisy if daily OS bugs cause trouble.